core/src/main/scala/tech/beshu/ror/es/BuildInfoReader.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.es
+
+import java.util.{Objects, Properties}
+
+final case class BuildInfo(esVersion: String, pluginVersion: String)
+object BuildInfoReader {
+  private val filename = "git.properties"
+
+  def create(filename: String = filename): BuildInfo = {
```

Comment:
this method could return `Try[BuildInfo]` ... let the caller decide what to do when there is no way to use BuildInfo object

Reply:
Now, I see that `Task[Buildinfo]` will be better type

Reply:
That makes sense.